test/e2e/csc_tests.go

```diff
@@ -56,9 +44,32 @@ func cscWithNonExistingTargetNamespace(t *testing.T, f *test.Framework, ctx *tes
 	}
 	err = createRuntimeObject(f, ctx, testOperatorSource)
 	if err != nil {
-		return err
+		t.Fatal(err)
 	}
 
+	// Run tests
```

Comment:
Resources that are created with ctx will not be deleted until the defer command is ran. This means that resources created in one test may affect another those in another test. Is this the approach that we want to take?